.mocharc.yml

```diff
@@ -1,7 +1,14 @@
-require: test/setup
-ui: bdd
+require: 'test/setup'
```

Comment:
the quotes are because YAML tries to be too smart. I like YAML, but for simple things only, and it must be used with caution.  quoting strings goes a long way.

Reply:
updated